### ISSUE TITLE
Matching breaks with 3.2.0 or newer

### ISSUE THREAD

**Issue by danez** [mrmlnc/fast-glob#273]
### Environment

  * OS Version: macOS 10.15.5
  * Node.js Version: 12.18.0

### Actual behavior (3.2.2)
```
fast-glob([
  '/path/to/src/modules/*/actions/**/*.php',
  '!**/src/modules/go/**/*.php',
])
[]
```

### Expected behavior (3.1.1)
```
fast-glob([
  '/path/to/src/modules/*/actions/**/*.php',
  '!**/src/modules/go/**/*.php',
])
[
/path/to/src/modules/A/actions/A.php",
"/path/to/src/modules/A/actions/B.php",
"/path/to/src/modules/A/actions/C.php",
"/path/to/src/modules/A/actions/D.php",
"/path/to/src/modules/A/actions/E.php",
"/path/to/src/modules/B/actions/sub/A.php",
 ... 2914 matches
]
```
### Steps to reproduce

  1. See above

### Code sample

```js
fast-glob([
  '/path/to/src/modules/*/actions/**/*.php',
  '!**/src/modules/go/**/*.php',
])
```

**Comment by mrmlnc** [comment]
Thank you for the detailed description of the issue :tada:

Based on the description, I think it's a duplicate for #266. I'll fix it soon.

**Comment by mrmlnc** [comment]
Shipped with [`3.2.3`](https://github.com/mrmlnc/fast-glob/releases/tag/3.2.3).

**Comment by danez** [comment]
Thanks works.
